### PR TITLE
Leverage the Karma package to run tests for web.

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: test/test.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+
+const config = require('bedrock').config;
+const path = require('path');
+
+config.karma.suites['web'] = path.join('web', '**', '*.js');
```
Comment (dlongley) [inline test/test.config.js]: Should this suite be `bedrock-web-store` instead? Why is it `web`?

**Review by dlongley — APPROVED** [review]
LGTM other than test suite name -- I left a comment on that. I'd like to get approval from @mattcollier here too since he's the Test Master.

File: test/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bedrock-web-store-test",
+  "version": "0.0.1-0",
+  "description": "Bedrock web store test",
+  "scripts": {
```
Comment (mattcollier) [inline test/package.json]: add `private` to test package files.  Prevents accidental NPM publishing from the wrong folder.

File: test/test.config.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+
+const config = require('bedrock').config;
```
Comment (mattcollier) [inline test/test.config.js]: es6

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: test/package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/bedrock-web-store"
+  },
+  "keywords": [
+    "bedrock"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "http://digitalbazaar.com"
```
Comment (davidlehn) [inline test/package.json]: https

File: test/web/10-sample.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      .to.throw(/No storage engine set./);
+
+    done();
+  });
+
+  it('Store should reject creating anything that is not an object', done => {
+    store.setEngine({engine: new MemoryEngine()});
+
+    expect(() => store.create({id: '123', object: 'foo'}))
+      .to.throw(/must be a object/);
```
Comment (davidlehn) [inline test/web/10-sample.js]: Change for code and test: a => an

**Review by mattcollier — APPROVED** [review]

**Pull request merged**